lib/mayaUsd/ufe/UsdUndoMaterialCommands.cpp

```diff
@@ -34,6 +35,45 @@ PXR_NAMESPACE_USING_DIRECTIVE
 namespace MAYAUSD_NS_DEF {
 namespace ufe {
 
+namespace {
+// We will not use the value of UsdUtilsGetMaterialsScopeName() for the material scope.
+static const std::string kDefaultMaterialScopeName("mtl");
```

Comment:
Is the reason for that eplained somwehere? half for my own understanding, half maybe the reason behind this comment should be added here.